hw/drivers/sensors/bma253/src/bma253.c

```diff
@@ -1065,10 +1317,15 @@ bma253_set_int_routes(const struct bma253 * bma253,
               (((int_routes->high_g_int_route & INT_ROUTE_PIN_2) != 0) << 1) |
               (((int_routes->low_g_int_route & INT_ROUTE_PIN_2) != 0) << 0);
 
+    rc = get_registers((struct bma253 *)bma253, REG_ADDR_INT_MAP_0, &regv, 1);
```

Comment:
does INT_MAP_0 and INT_MAP_1 need to be read before it is set? regv is never used for anything. INT_MAP_2 is not read before it is written